lanl/docker-compose/README.md

```diff
@@ -1,43 +1,114 @@
 # Deployment Recipe for use with Docker compose
 
-1. Run the `generate-creds.sh` script to generate a `.env` file populated with randomly-generated passwords for each Postgres database. This file will be read by Docker Compose.
-1. Run:
+## Deployment Files
+
+- `ochami-services.yml`: The main deployment file. Runs SMD and BSS with JWT
+  authentication enabled, plus postgres for SMD/BSS data storage and dnsmasq for
+  handling DHCP requests from nodes.
+- `ochami-services-noauth.yml`: Runs SMD and BSS with JWT authentication
+  disabled. Depends on the postgres container in `ochami-services.yml`.
+- `ochami-hurl-tests.yaml`: Runs integration tests using Hurl against the
+  authentication-enabled BSS and SMD in `ochami-services.yml`.
+- `ochami-hurl-tests-noauth.yml`: Runs integration tests using Hurl against the
+  authentication-disabled BSS and SMD in `ochami-services-noauth.yml`.
+- `ochami-krakend-ce.yml`: Runs the Krakend-CE API gateway for SMD and BSS.
+- `hydra.yml`: Runs the Hydra OAuth2/OIDC server used for authentication-enabled
+  BSS and SMD.
+
+## Running Deployments
+
+1. Run the `generate-creds.sh` script to generate a `.env` file populated with
+   randomly-generated passwords for each Postgres database. This file will be
+   read by Docker Compose.
+1. SMD and BSS support optional JWT authentication. `ochami-services.yml` runs
+   SMD and BSS with it enabled while `ochami-services-noauth.yaml` runs SMD and
+   BSS without it (although it depends on `ochami-services.yml` still).
+
+   To run the services with JWT authentication enabled:
 
    ```
    docker compose -f ochami-services.yml -f ochami-krakend-ce.yml up
    ```
 
-   to bring up ochami services.
+   **NOTE:** Authenticated BSS and SMD run on ports 27778 and 27779,
+   respectively.
+
+   To run both authenticated and non-authenticated services simultaneously:
+
+   ```
+   docker compose -f ochami-services.yml -f ochami-krakend-ce.yml -f ochami-services-noauth.yml -f hydra.yml up
```

Comment:
Does this run both auth/no-auth versions of SMD and BSS as well?

Reply:
Yes. `ochami-services.yml` runs BSS/SMD with auth enabled, `ochami-services.yml` runs both with auth disabled.

Reply:
I'm assuming you meant `ochami-services-no-auth.yml` runs both with auth disabled.

Reply:
Whoops, yes. Corrected.